graphics/include/gz/common/MeshManager.hh

```diff
@@ -240,6 +240,10 @@ namespace gz
                                const gz::math::Vector2d &_segments,
                                const gz::math::Vector2d &_uvTile);
 
+      /// \brief Set the assimp mode
+      /// \param[in] _useAssimp Set to true to enable assimp mode.
+      public: void SetUseAssimp(const bool _useAssimp);
```

Comment:
More descriptive doc
```suggestion
      /// \brief Set who is going to read the mesh file. This only work for Collada and OBJ files
      /// If True ASSIMP will load the file, otherwise the internal loader will do it
      /// \param[in] _useAssimp Set to true to load the mesh with ASSIMP.
      public: void SetUseAssimp(const bool _useAssimp);
```